IntroductionToVIPER.md

```diff
@@ -1,31 +1,31 @@
-**VIPER** - Architecture approach for application development(particularly iOS), based on ideas of [Robert Martin](http://blog.cleancoder.com/), for his article [The Clean Architecture](https://blog.8thlight.com/uncle-bob/2012/08/13/the-clean-architecture.html).
+**VIPER** - Architecture approach for application development(particularly iOS), based on [the Clean Architecture](https://blog.8thlight.com/uncle-bob/2012/08/13/the-clean-architecture.html) by [Robert C. Martin (Uncle Bob)](http://blog.cleancoder.com/).
 
 ![Clean Architecture](http://i.imgur.com/rt9bUjo.png)
 
 **Main goals of VIPER**:
 
 - Increase of test coverage for Presentation level, usually built from *Massive View Controllers*.
-- Partition of most heavy application classes into smaller parts with distinct responsibility.
+- Separate responsibility into the distinct layer.
```

Comment:
I think that it is more important of dispersion of responsibilities rather than breaking big size of classes to smaller elements.

`Separate responsibility into the distinct layer and classes` would be more clear.
Thoughts?